MigrationCodelab/build.gradle

```diff
@@ -25,7 +25,6 @@ buildscript {
         // App dependencies
         appCompatVersion = '1.4.2'
         composeCompilerVersion = '1.3.2'
-        composeVersion = '1.3.0'
```

Comment:
in the other codelabs you removed the dependency versions and added them directly in the gradle file. 
Should we do the same for this codelab?